src/components/OrdersWidget/index.tsx

```diff
@@ -206,7 +227,7 @@ const OrdersWidget: React.FC = () => {
             <div className="ordersContainer">
               <div className="headerRow">
                 <div className="checked">
-                  <input type="checkbox" />
+                  <input type="checkbox" onClick={toggleSelectAll} checked={orders.length === markedForDeletion.size} />
```

Comment:
It's more common to use `onChange` instead of `onClick` for checkboxes and radio elements. I can't say if one is better than the other, just more common